addons/sale_mrp/models/sale.py

```diff
@@ -127,6 +127,12 @@ def _compute_qty_delivered(self):
                     else:
                         order_line.qty_delivered = 0.0
 
+    def compute_uom_qty(self, new_qty, stock_move, rounding=True):
+        #check if stock move concerns a kit
+        if stock_move.bom_line_id:
+            return new_qty * stock_move.bom_line_id.product_qty
+        return super(SaleOrderLine, self).compute_uom_qty(new_qty, stock_move, rounding)
```

Comment:
This is getting slowly changed in Odoo
```suggestion
        return super().compute_uom_qty(new_qty, stock_move, rounding)
```